webapp/publisher/snaps/views.py

```diff
@@ -1464,3 +1464,34 @@ def post_trigger_build(snap_name):
     return flask.redirect(
         flask.url_for(".get_snap_builds", snap_name=snap_name)
     )
+
+
+@publisher_snaps.route("/<snap_name>/builds/validate-repo", methods=["GET"])
```

Comment:
Please create a new file`webapp/publisher/snaps/views_build.py` (like this [one](https://github.com/canonical-web-and-design/snapcraft.io/blob/master/webapp/store/snap_details_views.py)) and move this function in there. 
You will be able to start moving the others related to build later.

Reply:
Hey Toto,

You're right I've been telling you that I will do it... I just moved only this route to a new file for now.